src/ui/SearchInterface/SearchInterface.ts

```diff
@@ -478,6 +479,8 @@ export class SearchInterface extends RootComponent implements IComponentBindings
     $$(this.element).on(QueryEvents.querySuccess, (e, args) => this.handleQuerySuccess(args));
     $$(this.element).on(QueryEvents.queryError, (e, args) => this.handleQueryError(args));
 
+    this.queryStateModel.registerNewDynamicAttribute(QueryStateModel.attributesEnum.fv);
```

Comment:
Yeah, let's say I was not quite sure about this solution. I found it cleaner in the URL to have `fv:@objecttype=[message]` than `fv={"@objecttype": "[message]"}`, and it was closer to what the facets look like. But since no component was registering `fv:@objecttype` (and no component would, by design...), it was being rejected from the state, hence the `dynamic attribute` thing.

I can change to a simple JSON state if you would rather have that :)

Reply:
I think it would be better if we did not have to add a new concept, yes. It should be simpler to maintain, in theory.

Any opinion @dms1lva @samisayegh  ?